internal/agent/list.go

```diff
@@ -56,36 +62,92 @@ func NewAgentList(loader func(int) tea.Cmd, page, perpage int) AgentListModel {
 }
 
 func (m *AgentListModel) appendAgents() tea.Cmd {
+	return m.loadAgents(false)
+}
+
+func (m *AgentListModel) refreshAgents() tea.Cmd {
+	m.isRefreshing = true
+	return m.loadAgents(true)
+}
+
+func (m *AgentListModel) loadAgents(isRefresh bool) tea.Cmd {
```

Comment:
Looks like it could append indefinitely in long-running sessions? Is there a limit or pagination somewhere?